src/Autosuggest.js

```diff
@@ -581,7 +583,12 @@ export default class Autosuggest extends Component {
 
             break;
 
+          // Tab (must fall through to enter when conditions apply!)
+          case 9:
+            if (!this.props.selectOnTab) break;
```

Comment:
IMO, I believe Tabbing is heavily used as an "autocomplete" feature. That being said I feel that tabbing should automatically select the first item in the suggestions list if none are highlighted. "Enter", however should continue to remain dependent on a highlighted suggestion

Reply:
I don't. I would suggest that using the up/down arrow key to select then tab selects the item.

Reply:
Hello guys, 
I will migrate to this component at the moment, and I've tested the current version with screen readers. The long story made short, the suggestions are not read. I will most likely prepare a PR to improve it in that sense. 

As of the keyboard requirements for accessibility, the UX rules normally are:
Keyboard Interaction

* UP and DOWN: Cycles through auto-suggestions and input field.
* ESC: Close the list with options
* ENTER: Select the currently focused auto-suggestion and close the menu.
* TAB: Select the currently focused auto-suggestion, close the menu, and move focus to the next focusable element.

I hope this helps to figure out the expected behaviour